src/main/java/com/internet/mixshop/controller/CartController.java

```diff
@@ -0,0 +1,31 @@
+package com.internet.mixshop.controller;
+
+import com.internet.mixshop.lib.Injector;
+import com.internet.mixshop.model.Product;
+import com.internet.mixshop.model.ShoppingCart;
+import com.internet.mixshop.service.ShoppingCartService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class CartController extends HttpServlet {
+    private static final Long USER_ID = 1L;
+
```

Comment:
we don't need here empty line, while declaring fields

Reply:
done